### ISSUE TITLE
Spurious unused binding warning when passing a binding to alias

### ISSUE THREAD

**Issue by aviflax** [borkdude/clj-kondo#649]
**version**

v2019.11.23

**platform**

Native image installed via Homebrew on MacOS 10.14.

Java:

```
openjdk version "11.0.4" 2019-07-16
OpenJDK Runtime Environment AdoptOpenJDK (build 11.0.4+11)
```

**editor**

I was able to reproduce this problem from the command line

**problem**

This code is correct:

```clojure
(doseq [[ns-sym _ alias-sym] (cons t ts)]
  (create-ns ns-sym)
  (alias alias-sym ns-sym))
```

but clj-kondo issues a warning about the first line: `warning: unused binding alias-sym`

**repro**

```clojure
(ns repro)

(doseq [[ns-sym _ alias-sym] [['foo :as 'f]]]
  (create-ns ns-sym)
  (alias alias-sym ns-sym))
```

```shell
$ clj-kondo --lint /tmp/repro.clj
/tmp/repro.clj:3:19: warning: unused binding alias-sym
linting took 9ms, errors: 0, warnings: 1
```

**expected behavior**

I would expect no warning.
